assets/scripts/photo/events.js

```diff
@@ -20,9 +20,20 @@ const onIndexPhotos = function (event) {
     .catch(ui.onIndexPhotosFailure)
 }
 
+const onShowPhoto = function (event) {
+  event.preventDefault()
+
+  const id = $(event.target).data('id')
+  api.showPhoto(id)
+    .then(ui.onShowPhotoSuccess)
+    .catch(ui.onShowPhotoFailure)
+}
+
 const addHandlers = function () {
   $('#photo-create').on('submit', onCreatePhoto)
   $('#photo-index').on('click', onIndexPhotos)
+  $('#index-wrapper').on('click', 'h3', onShowPhoto)
+  $('#index-wrapper').on('click', 'img', onShowPhoto)
```

Comment:
These should probably have a single class that we are using for this event handler. Would avoid having issues with other h3/img elements later, and also let us combine this into one event handler.